src/labels/CustomLabels.labels

```diff
@@ -1753,6 +1753,14 @@
         <shortDescription>Enablement section warning</shortDescription>
         <value>&lt;b&gt;WARNING: NPSP doesn't provide a way to revert back to legacy Recurring Donations.&lt;/b&gt;</value>
     </labels>
+    <labels>
+        <fullName>RD2_EnablementIncomplete</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Enhanced Recurring Donations enablement incomplete</shortDescription>
+        <value>You cannot create or update Recurring Donations because you haven't finished enabling Enhanced Recurring Donations. Go to NPSP Settings | Recurring Donations | Upgrade to Enhanced Recurring Donations and complete all enablement steps.</value>
```

Comment:
```suggestion
        <value>You cannot create or update Recurring Donations because you haven&apos;t finished enabling Enhanced Recurring Donations. Go to NPSP Settings | Recurring Donations | Upgrade to Enhanced Recurring Donations and complete all enablement steps.</value>
```